Added docs for the error message changes (#26687)
hashicorp/vault: website/content/docs/concepts/tokens.mdx

As a corollary, batch tokens can be used across performance replication
clusters, but only if they are orphan, since non-orphan tokens will not be able
to ensure the validity of the parent token.

## Error Responses

> When using a token that has been revoked, exceeded it's TTL, or is an otherwise invalid value, Vault will respond

Reviewer: ```suggestion
When using a token that has been revoked, exceeded its TTL, or is an otherwise invalid value, Vault will respond
```

Reviewer: "It's" means "it is", whereas "its" is the possessive (like "hers")

Author: oooops!!

> with a 403 error containing the following error messages: `invalid token` and `permission denied`.

Reviewer: ```suggestion
with a `403` response code error containing the following error messages: `invalid token` and `permission denied`.
```

> If using a token that with incorrect policy access, Vault will respond with a 403 error containing the error message

Reviewer: ```suggestion
If using a token with incorrect policy access, Vault will respond with a `403` response code error containing the error message
```

Reviewer: Reopened as a different suggestion
